appinventor/components/src/edu/mit/appinventor/ai/posenet/PosenetExtension.java

```diff
@@ -376,6 +377,11 @@ public YailList RightAnkle() {
     return keyPoints.get("rightAnkle");
   }
 
+  @SimpleProperty(description = "BackGround Image.")
+  public String backgroundImage() {
```

Comment:
To be consistent with the naming of blocks, I would recommend that this be `BackgroundImage`